### DISCUSSION TITLE
分享真．AI Vtuber

### DISCUSSION THREAD

**Discussion by christaaah** [1101-GS4719-C/Python-Community-GS4719-C#2205] [General]
想分享一下之前很喜歡的Vtuber，她的名字叫Neuro sama，如標題所說，她是一位真正的AI Vtuber，之所以會這麼說，是因為有角色設定是人工智慧但是中之人卻是真人的（絆愛）。Neuro sama的遊戲AI是由python來運行的，其他聊天對話是C#完成的。如果事先不知道她是AI，去看她的直播會以為她是真人扮演的，因為她的反影時間非常的短，回覆時間不到三秒，並且了解語句中的雙關、反諷等等更深的涵義，還會有些微的語氣變化，讓其更像真人。

**Comment by 112121py** [comment]
推版主，本來也想發的w
Neuro真的很有趣，在你知道他是AI的前提下，看他跟他的開發者Vedal或是其他實況主連動，那個對話然後超級會做效果還會亂嗆xd，Vedal也會開開發直播，會一邊更新他女兒的功能一邊跟大家聊天，不管對Vtuber有沒有興趣的都可以去看看，youtube上也蠻多大家剪的精華的，都很有綜藝效果ww 總之超酷，推一個

**Comment by poyen0806** [comment]
推，之前還有同學在英文課報告Neuro。youtube上演算法偶爾會推薦給我別人剪輯的精華，很多部分都挺好笑的，尤其是與人的互動上，其與其開發者vedel的對話更是有趣，想像自己開發出來的ai一直在嗆自己，很難不覺得有趣吧，推

**Comment by Boying12** [comment]
推推，她能夠理解語句中的雙關語和反諷，讓我感受到她的智慧和幽默。她的語氣變化也讓她更具人性化，讓人幾乎忘記她是一個AI。這種互動模式讓我在觀看她的直播時，感覺像是在與真正的朋友對話，而不僅僅是看一個虛擬角色。
